SemanticAutocomplete.md

```diff
@@ -0,0 +1,128 @@
+
+For ease of use DSAD autogenerates and publishes Java and C# SDKs for all of our services using swagger. For more information on usage please see our SDK repo [here](https://github.com/cbdr/dsad-sdks).
+
+## Request
+As with all DSAD services, Semantic Autocomplete accepts requests sent as either HTTP GET or HTTP POST. Access to semantic autocomplete is made available at:
+ https://www.api.careerbuilder.com/core/semanticsearch/autocompleterequires
+
+ This service requires Careerbuilder Oauth 2.0 client credentials. For more information please see our ReadMe [here](/Readme.md#access).
+
+#### Parameters:
+| Parameter | Require/Optional | Type | Description |
+|-----------|------------------|------|-------------|
+| query | Required | String | The `query` value to autocomplete.
+| limit | Optional | Integer | The number of results expected to return.
+
+**The limit field cannot exceed a size of 100 and defaults to 10 without the value present.**
+
+The current version of the service is `2.0`. Version must be specified in the ```Accept``` header.
+
+Example: ```Accept: application/json;version=2.0```
+
+## Response
+The response data is broken up into two parts: the original `query` and the list of suggestions returned from Solr. The first `suggestion` will always be identified with the `type` *free_text* and the associated `value` is that of  the highest `popularity` of the returned results.
+
+### Data
+| Field | Type | Description |
+|-------|------|-------------|
+| query | String | Original query sent to Solr.
+| suggests | Suggestion[] | A json array of Suggestions broken up by type.
+
+### Suggestion
+
+| Field | Type | Description |
+|-------|------|-------------|
+| type | String | The type or cannonical form of the suggestion. Returned values include: *keyword, job_title, job_level, occupation, skill, school, company, location, freetext*
+| values | value[] | A json array of values associated with the type.
+
+
+### Value
+A value object represents the surface forms associated with the type(cannonical) suggested for autocomplete. E.g., provided "e" as a query, under the `type` *job_title* you would see `names` like engineer, executive, sales executive, etc.
+
+| Field | Type | Description |
+|-------|------|-------------|
+| name | String | A name associated with the type classification.
+| popularity | Integer | An integer value representing the score of popularity indicating how often the value is used.
+
+#### Notable missing fields
+The older service maintained a timing object as part of the data field. This is no longer returned.
```

Comment:
Instead of 'The older service' I would say "Version 1 of this services returned a timing..."